testdata/project-v3/go.mod

```diff
@@ -10,4 +10,5 @@ require (
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v0.19.2
 	sigs.k8s.io/controller-runtime v0.7.0
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20210113160450-b84d99da0217
```

Comment:
@Adirio,

The project file is missing the info over the --plugin=declarative used to create the api for project-v2 and project-v3.

Reply:
Done